aperturedb/Descriptors.py

```diff
@@ -0,0 +1,150 @@
+import logging
+
+import numpy as np
+
+from aperturedb.Entities import Entities
+
+logger = logging.getLogger(__name__)
+
+
+class Descriptors(Entities):
+    """
+    Python wrapper for ApertureDB Descriptors API.
+    """
+
+    db_object = "_Descriptor"
+
+    def __init__(self, db, response=None):
```

Comment:
The response, in Images is to facilitate a easy wrapper creation when handling a response from a JSON query. The helper methods become easy to use then. This is obviously not interface specific, but will be a good to consider.

Reply:
OK.  I think that would have worked, but I don't think it's as useful for descriptors, so I took it out.